.gitignore

```diff
@@ -7,3 +7,4 @@ models/iris/*.sdf
 *.pyc
 .*.sw?
 *-gen.sdf
+models/water_level/*
```

Comment:
Why do we need to remove this?

Reply:
![image](https://user-images.githubusercontent.com/10233412/94296227-839f7500-ff30-11ea-8470-61923ca30143.png)

Reply:
@bperseghetti I am mostly sure you have a copy paste problem on your folder

Reply:
I assume we don't want all of that in the repo...

Reply:
Look at the folder structure

Reply:
Yeah, it looks like the models were all copied into it.

Reply:
Good catch, on clean build from inside of Firmware tools after forcing sub-module over to my fork there appears to be no issue. Oh the joys of needing to work on forked sub-modules. Reverting.